CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 ### Fixed
 
 - Fixes [#3344](https://github.com/gitkraken/vscode-gitlens/issues/3344) - Make changing the AI key easier
+- Fixes [GLVSC-569] - refresh Jira token of an integration on GK
```

Comment:
We shouldn't reference internal Jira tickets in the public changelog. Something less specific, like "fixes issue with Jira integration not refreshing", should suffice.

Reply:
fixed